packages/js/product-editor/src/components/section-header/section-header.tsx

```diff
@@ -46,10 +45,10 @@ export function SectionHeader( {
 				</h2>
 
 				<div className="wp-block-woocommerce-product-section-header__actions">
-					<BlockSlot name={ `${ type }-actions` } />
+					<BlockSlot name={ `section-actions` } />
 				</div>
 			</div>
-			<BlockSlot name={ `${ type }-description` } />
+			<BlockSlot name={ `section-description` } />
 		</HeadingTagName>
```

Comment:
Default the section header slots to use `section-` prefix instead of `subsection-` and `section-`, this is because a specific block won't know if it is nested underneath a subsection or section, this means that with the above they did always have to add both slot fills which seems redundant to me.